src/pyhf/infer/test_statistics.py

```diff
@@ -47,16 +89,88 @@ def qmu(mu, data, pdf, init_pars, par_bounds):
         raise UnspecifiedPOI(
             'No POI is defined. A POI is required for profile likelihood based test statistics.'
         )
+    if par_bounds[pdf.config.poi_index][0] == 0:
+        log.warning(
+            'qmu test statistic used for fit configuration with POI bounded at zero. Use qmutilde.'
+        )
+    return _qmu_like(mu, data, pdf, init_pars, par_bounds)
 
-    tensorlib, optimizer = get_backend()
-    mubhathat, fixed_poi_fit_lhood_val = fixed_poi_fit(
-        mu, data, pdf, init_pars, par_bounds, return_fitted_val=True
-    )
-    muhatbhat, unconstrained_fit_lhood_val = fit(
-        data, pdf, init_pars, par_bounds, return_fitted_val=True
-    )
-    qmu = fixed_poi_fit_lhood_val - unconstrained_fit_lhood_val
-    qmu = tensorlib.where(
-        muhatbhat[pdf.config.poi_index] > mu, tensorlib.astensor(0.0), qmu
-    )
-    return tensorlib.clip(qmu, 0, max_value=None)
+
+def qmu_tilde(mu, data, pdf, init_pars, par_bounds):
+    r"""
+    The test statistic, :math:`\tilde{q}_{\mu}`, for establishing an upper
+    limit on the strength parameter, :math:`\mu` for models with
+    bounded POI.
+
+    Args:
+        mu (Number or Tensor): The signal strength parameter
+        data (Tensor): The data to be considered
+        pdf (~pyhf.pdf.Model): The HistFactory statistical model used in the likelihood ratio calculation
```

Comment:
```suggestion
        pdf (~pyhf.pdf.Model): The statistical model adhering to the schema model.json
```
As we're this line is getting changed then maybe we can take this opportunity to unify the descriptions.